src/main/java/com/oneops/secrets/command/SecretUpdate.java

```diff
@@ -79,6 +79,9 @@ public void exec() {
                         .append(yellow(dot(String.format("Applications can access secret content by reading '/secrets/%s' file.", secretName))))
                         .append(lineSep)
                         .append("  ")
+                        .append(yellow(dot(String.format("Secrets can be accessed only by the User/Group configured in 'secrets-client' component in '%s' env.", app.getNsPath().toLowerCase()))))
```

Comment:
@sureshg can we create some message text constants like for the 2 similar text messages in this case

Reply:
yeah.. secrets add/update follow-up messages are differed by few lines/words. Will extract the error messages in `v1.0.3`

Reply:
@sureshg ok, merging